tests/fixtures/models/question1/config.yaml

```diff
@@ -0,0 +1,30 @@
+expectation_features:
+- bad_regex:
+  - \brule\w*\b.*\bdiffer\w*\b
+  good_regex:
+  - \b(group|peer|lead\w*|command\w*)\b.*\b(pressu\w*|influ\w*)\b
+  ideal:
+  - peer
```

Comment:
I know at runtime the ideal sentence needs to be tokenized, but it's awkward to have it that way in the config. Is there any reason we can't have a normal string for the ideal sentence here and then tokenize it when we need to?

e.g.

```yaml
    ideal: peer pressure can cause inappropriate behavior
```

...I know the actual words in the current examples are slightly different. We should discuss if they really can't be more natural english sentences, because remember that in practice, most of these ideal sentences are going to come from teachers creating them in that web lesson editor

Reply:
As discussed, a teacher won't change anything inside the models/ folder. So this won't create any problem.

Reply:
ok sounds good